pyxem/generators/calibration_generator.py

```diff
@@ -334,6 +336,78 @@ def get_navigation_calibration(self, line_roi, x1, x2, n, xspace,
 
         return x[0]
 
+    def get_rotation_calibration(self, real_line_roi, reciprocal_line_roi):
+        """Determine the rotation between real and reciprocal space coordinates.
+
+        Parameters
+        ----------
+        real_line_roi : Line2DROI
+            Line2DROI object drawn along known direction in real space.
+        reciprocal_line_roi : Line2DROI
+            Line2DROI object drawn along known direction in reciprocal space.
+
+        Returns
+        -------
+        rot_cal : float
+            Rotation angle in degrees.
+        """
+        # Calculate real space vector from line roi
```

Comment:
`Line2DROI` has a `.angle() ` method, so I think you could just do something like
`self.rotation_angle = real_line_roi.angle() - reciprocal_line_roi.angle()`

saves a bunch of lines, and a util.

Reply:
Thanks, will do that.